test/TestAppCX/TestAppCX.vcxproj

```diff
@@ -124,6 +124,11 @@
     <Import Project="$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props" Condition="exists('$(UserRootDir)\Microsoft.Cpp.$(Platform).user.props')" Label="LocalAppDataPlatform" />
   </ImportGroup>
   <PropertyGroup Label="UserMacros" />
+  <ItemDefinitionGroup>
+    <ClCompile>
+      <ControlFlowGuard Condition="'$(Configuration)'=='Release'">Guard</ControlFlowGuard>
```

Comment:
What does this do?

Reply:
This enables ControlFlowGuard which is a security feature. Every time a virtual call is made, the address of the function is validated against a known good list of functions. It prevents jumping to an invalid location. This is one of the things that BinSkim checks for. Since this is only a test app, this flag is less important. But due to how the template works, there is no easy way to scan only certain files and ignore others. So it is easier to just enable this flag in the test apps also.